src/Types/Transaction.purs

```diff
@@ -875,6 +876,13 @@ instance Show DataHash where
 -- To help with people copying & pasting code from Haskell to Purescript
 type DatumHash = DataHash
 
+newtype BlockId = BlockId ByteArray
```

Comment:
Isn't this the same as the `BlockHeaderHash`? Maybe we could deduplicate the the types (the other one is a `String`, but perhaps should be a `ByteArray`). We'd just need to adjust the `getChainTip` return type